app/tests/access/test_access_api.py

```diff
@@ -0,0 +1,609 @@
+from unittest.mock import patch
+
+from access.api import user_to_response
+from access.models import User
+from access.schemas import UserSchema
+from botocore.exceptions import EndpointConnectionError
+from provider.models import Provider
+from pytest import fixture
+
+
+@fixture(name='access_data')
+def fixture_access_data(db):
```

Comment:
The name `access_data` confuses me a bit. I thought this data was for testing the "access", but below it seems to be about the user in the db to run the tests on

Reply:
OK so it's access data because this here is the tests for the access app? Maybe add a little docstring to the fixture describing the purpose of it